mote/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 import flask, random, string, json, util, re
 import dateutil.parser, requests, collections
-import logging
+import logging, soke
```

Comment:
We should definitively keep the import since otherwise we are creating an error inside a exception handling :)